labelme/app.py

```diff
@@ -1601,23 +1601,40 @@ def enableKeepPrevScale(self, enabled):
     def onNewBrightnessContrast(self, qimage):
         self.canvas.loadPixmap(QtGui.QPixmap.fromImage(qimage), clear_shapes=False)
 
-    def brightnessContrast(self, value):
+    def brightnessContrast(self, value: bool, is_initial_load: bool = False):
+        del value
+
         dialog = BrightnessContrastDialog(
-            utils.img_data_to_pil(self.imageData),
+            utils.img_data_to_pil(self.imageData).convert("RGB"),
```

Comment:
This will lose transparency in the image. Wouldn't it be better to extract the RGB bands, run brightness/contrast on them, then splat in the original alpha?

Reply:
yes, that sounds better. https://github.com/wkentaro/labelme/issues/1660